c/src/platform/platform.h

```diff
@@ -81,7 +81,7 @@ int pni_vsnprintf(char *buf, size_t count, const char *fmt, va_list ap);
 
 #endif
 
-#if defined _MSC_VER || defined _OPENVMS
+#if defined _MSC_VER || defined(_WIN32) || defined _OPENVMS
```

Comment:
I am surprised this is needed. I always believed mingw is a version of GCC, and therefore there wouldn't be a version of mingw that does not define the va_copy macro.

It would be nice to switch to Visual Studio 2015+, and stop supporting 2013. Then there would be va_copy always defined even on MSVC.

Reply:
Note to myself: The switch I mentioned in previous comment already happened, so va_copy on Windows can be relied on.